app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  after_initialize :update_comments_counter
+  belongs_to :user, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments_counter.to_i + 1)
+  end
```

Comment:
- [ OPTIONAL] I recommend keeping the update comments  counter method private since you used one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_commet_counter` method every time a new comment has been saved to database, so I think it would be better if you keep as you make It first 💪🏻✅👏🏻💯🔥